File: CodeWars/.js/8kyu-Switch-it-Up.js
// When provided with a number between 0-9, return it in words.

// Input :: 1

// Output :: "One".

// If your language supports it, try using a switch statement.


const chai = require("chai");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("Basic Tests",() =>{
  it("Testing for fixed tests", () => {
    assert.strictEqual(switchItUp(1),"One");
    assert.strictEqual(switchItUp(3),"Three");
    assert.strictEqual(switchItUp(5),"Five");
  });
});


// my code

function switchItUp(number){
    switch (number){
        case 0:
          return 'Zero'
          break
        case 1:
          return 'One'
          break
        case 2:
          return 'Two'
          break
        case 3:
          return 'Three'
          break
        case 4:
          return 'Four'
          break
        case 5:
          return 'Five'
          break
        case 6:
          return 'Six'
          break
        case 7:
          return 'Seven'
          break
        case 8:
          return 'Eight'
          break
        case 9:
          return 'Nine'
          break
    }
}

File: CodeWars/.cs/8kyu-SwitchItUp.cs
// When provided with a number between 0-9, return it in words.

// Input :: 1

// Output :: "One".

// If your language supports it, try using a switch statement.

namespace Solution
{
  using NUnit.Framework;
  using System;
  
  [TestFixture]
  public class KataTests
  {
    [Test]
    public void BasicTests()
    {
      Assert.AreEqual("One", Kata.SwitchItUp(1));
      Assert.AreEqual("Three", Kata.SwitchItUp(3));
      Assert.AreEqual("Five", Kata.SwitchItUp(5));
    }
  }
}

// my code

public class Kata
{
  public static string SwitchItUp(int number)
  {
    string returnString = "";
    switch (number)
    {
      case 1:
        returnString = "One";
        break;
      case 2:
        returnString = "Two";
        break;
      case 3:
        returnString = "Three";
        break;
      case 4:
        returnString = "Four";
        break;
      case 5:
        returnString = "Five";
        break;
      case 6:
        returnString = "Six";
        break;
      case 7:
        returnString = "Seven";
        break;
      case 8:
        returnString = "Eight";
        break;
      case 9:
        returnString = "Nine";
        break;
      case 0:
        returnString = "Zero";
        break;
    }
    return returnString;
  }
}